02-s3/README.md

```diff
@@ -310,6 +336,8 @@ through the CLI or the console.
 _Can you change a single tag on a bucket or object, or do you have to change
 all its tags at once?_
 
+> A: It looks like you can do either:`aws s3api put-bucket-tagging` and `aws s3api put-object-tagging` 
+
```

Comment:
The S3 tagging API manages the entire set of tags in a single API call. See the [S3 Object Tagging](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-tagging.html) doc:

PUT Object tagging replaces tags on an object…. To modify the existing tag set, you must first retrieve the existing tag set, modify it on the client side, and then use this API to replace the tag set. If you send this request with empty tag set, S3 deletes existing tag set on the object.